escapades_backend/Gemfile

```diff
@@ -37,7 +37,7 @@ gem "rack-cors"
 
 group :development, :test do
   # See https://guides.rubyonrails.org/debugging_rails_applications.html#debugging-with-the-debug-gem
-  gem "debug", platforms: %i[ mri windows ]
+  # gem "debug", platforms: %i[ mri windows ]
```

Comment:
I would replace it by [pry-byebug](https://github.com/deivid-rodriguez/pry-byebug)